packages/constellation/src/components/Compositions/Input/Input.types.ts

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react'
+
+type InputProps = {
```

Comment:
I don't see the value, onChange, onFocus.... attributes that you see in the React input type

Maybe you could use the HTMLInputAttributes from react here?